.github/workflows/versions.yml

```diff
@@ -0,0 +1,368 @@
+name: versions
+on:
+  - pull_request
+  - push
+
+jobs:
+  verify-tags:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Verify Setup For Tags
+        run: |
```

Comment:
Instead of having an individual entry for each version, I would suggest using a [job matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs)